### DISCUSSION TITLE
regarding issues in GitHub Tshirt Gift Code

### DISCUSSION THREAD

**Discussion by mridungupta1984** [community/community#49639] [GitHub Education] [answered]
The coupon that Imprint Engine is assigning to the request made under GitHub Teacher Toolbox Swag Program are found to be having some glitches that they are not usable on [The GitHub Shop](https://www.thegithubshop.com/). I request someone from @github-staff or the Imprint Engine team to look into this.
---
_Error: The coupon code is invalid Enter a valid discount code or gift card
Error: This discount has reached its usage limit_

> ```The coupon is unused from the date I received them. This issue is not only with me, but also with my fellow teachers also.```

Already, send a mail to Imprint Engine vide number #276747, and also made a ticket vide number #2035786

**Comment by ghost** [comment]
I'm sorry to hear that you and other teachers are experiencing issues with the gift codes provided by Imprint Engine for the GitHub Teacher Toolbox Swag Program. It can be frustrating when the coupon codes don't work as expected, especially when they are being used for a good cause like education.

It's good to hear that you have already reached out to Imprint Engine and made a support ticket. They should be able to investigate the issue and provide a resolution soon. If you haven't already, I suggest following up with them to ensure that your ticket is being worked on.

In the meantime, you can also try reaching out to GitHub support via their official channels to report the issue and see if they can assist you in any way. They may be able to escalate the issue with Imprint Engine and help you resolve it more quickly.

I hope that the issue is resolved soon and that you and other teachers can receive the swag you were promised without any further issues.

**Reply by mridungupta1984** [reply]
Thank you, @MrShadowDev.

**Reply by ghost** [reply]
You're welcome! I'm glad I could help. Don't hesitate to reach out if you have any other questions or concerns in the future.

**Comment by samus-aran** [comment, marked as answer]
Hi @mridungupta1984  👋

We like to thank you for your feedback and sharing your experience of the GitHub Education Teachers swag program. Unfortunately GitHub t-shirts are currently not part of the [GitHub Teachers swag](https://education.github.com/teachers) offering and as a result, these coupons are no longer available for distribution.

We understand that this news may be disappointing, and we want to express our sincere apologies for any inconvenience this may have caused. Please know that this decision was not made lightly and we’ll continue to revise our Teacher swag offerings informing our educators of any updates in the future.

Once again, we apologize for any inconvenience this may cause and thank you for your continued support and participation in GitHub Education Global Campus.

**Reply by mridungupta1984** [reply]
These coupons were received to me before the revamping was scheduled, and from the same moment, I am trying to contact Imprint Engine and GitHub Support, so that I can get assisted at the earliest.

**Reply by mridungupta1984** [reply]
> These coupons were received to me before the revamping was scheduled, and from the same moment, I am trying to contact Imprint Engine and GitHub Support, so that I can get assisted at the earliest.

@samus-aran is not there any resolution for this case, as I am unable to use these coupons before the scheduled revamping.

**Reply by samus-aran** [reply]
Hi @mridungupta1984 we are unable to process any t-shirts as they are not part of the Teacher Swag program. They had been previously but have since been discontinued. We believe the reference to t-shirt coupons in your Teacher Swag receipt/order was in error. Again we apologise for the confusion and inconveniences this has caused.

**Reply by mridungupta1984** [reply]
> Hi @mridungupta1984 we are unable to process any t-shirts as they are not part of the Teacher Swag program. They had been previously but have since been discontinued. We believe the reference to t-shirt coupons in your Teacher Swag receipt/order was in error. Again we apologise for the confusion and inconveniences this has caused.

Thank you for the clarification, @samus-aran. May you provide a teaser or a hint of the upcoming offerings?

**Comment by Sir-karan-singh** [comment]
Even my order has t-shirt coupons. but I didn't receive them!

**Reply by turlakriju** [reply]
@fishmandev

**Comment by ghost** [comment]
@samus-aran any update sir?
